hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetInfoHandler.java

```diff
@@ -151,6 +151,7 @@ private Optional<FileInfo> infoForFile(
                         file.deleted());
             }
         } else {
+            requireNonNull(fileID);
```

Comment:
This is unnecessary, you cannot reach this line if fileID is null.

Reply:
fixed thanks